include/SFML/Graphics/Transform.hpp

```diff
@@ -409,6 +427,7 @@ SFML_GRAPHICS_API Vector2f operator *(const Transform& left, const Vector2f& rig
 ///
 /// Performs an element-wise comparison of the elements of the
 /// left transform with the elements of the right transform.
+/// Matrices are expected to be exactly equal; no epsilon is involved.
```

Comment:
Kind of unrelated to the related ticket and I think it might need additional clarification. While people who would look for such a clarification will be familiar with what "no epsilon is involved" really means, those who don't know the meaning, will be more confused than helped by this comment.

Might more useful to explain that due to floating point imprecision the equality might not always be fulfilled and potentially offer a solution for said problem.

Reply:
Thinking more about it, the main thing to express would be:

> Just because two matrices transform a vector in the same way, they need not be equal.

This can be due to floating point inaccuracy, but also going into the math details might be out of scope. I'm also not aware of a one-fits-all way to provide such a comparison. I removed the sentence for now, but we can gladly add a formulation if we find a good, intuitive one 🙂